### ISSUE TITLE
viewWillAppear  called for all  view controller after selection 1st to 5th controller

### ISSUE THREAD

**Issue by Sahilberi** [HighBay/PageMenu#252]
HI,
     When i go to 1st view controller to 4th or 5th all controllers  methods(viewWillAppear and viewDidAppear) in the  middle  get called which is shouldn't called.
 
It makes no sense to get called of middle controller methods if i tap to another controller menu item. only tapped menu item  controller methods should called.

**Comment by Luccifer** [comment]
Same problem... Any help to clear memory or just skip other controllers?

**Comment by hideki-a** [comment]
I have the same problem too.

When a comment out from L814 to L825 at CAPSPageMenu.swift, I was canceled by the problem.
https://github.com/uacaps/PageMenu/blob/master/Classes/CAPSPageMenu.swift#L814-L825

I don't know what kind of influence there is by this.
But, it seems to be running with no problem.

**Comment by MatthewYork** [comment]
I agree, that is inappropriate behavior. We will be looking into this.

**Comment by toseefkhilji** [comment]
@MatthewYork : Did you get chance to look into this?
